ui/packages/catalog-ui-search/src/main/webapp/component/navigator/navigator.hbs

```diff
@@ -31,6 +31,9 @@
     <button class="navigation-choice is-neutral choice-search-forms is-button" data-fragment="forms">
         <span class="fa fa-file-o"></span><span>Search Forms</span><div class="forms-indicator"></div>
     </button>
+    <button class="navigation-choice is-neutral choice-result-forms is-button" data-fragment="resultForms">
```

Comment:
❓ If we're naming this data-fragment `resultForms`, should we rename the one above `searchForms`?